react/package.json

```diff
@@ -20,6 +20,7 @@
     "myvtex-sse": "^1.2.0",
     "prop-types": "^15.6.0",
     "qs": "^6.5.1",
+    "ramda": "^0.25.0",
```

Comment:
We shouldn't add `ramda` to the runtime since it's pretty heavy. Or at least that's what we thought up until now. @brunoabreu what do you think?